src/kernels/kernel.node.ts

```diff
@@ -72,11 +72,11 @@ export class Kernel extends BaseKernel {
         if (getAssociatedNotebookDocument(this)?.notebookType === InteractiveWindowView) {
             // If using ipykernel 6, we need to set the IPYKERNEL_CELL_NAME so that
             // debugging can work. However this code is harmless for IPYKERNEL 5 so just always do it
-            if (await this.fs.localFileExists(AddRunCellHook.ScriptPath)) {
-                const fileContents = await this.fs.readLocalFile(AddRunCellHook.ScriptPath);
+            if (await this.fs.localFileExists(this.scriptPaths.addRunCellHook.scriptPath.fsPath)) {
+                const fileContents = await this.fs.readLocalFile(this.scriptPaths.addRunCellHook.scriptPath.fsPath);
```

Comment:
To make `scriptPathUtils` less Node specific, I made `scriptPath` into a `Uri`, and here I'm using `fsPath`.

Reply:
Maybe you could move the reading of scripts into scriptPathUtils? Then it's isolated in one spot.

Reply:
I'd like to think about that in a separate PR. This PR is pretty scoped. Adding `fs` to the `scriptPathUtils` is something I'd like to think with the greater context of changes I'm doing.

Reply:
I would disagree, merging this PR as is breaks this feature completely (i.e. we're going to knowingly introduce a bug), 
The way i would see this PR is `Ready the script utils for web`
Hence, I think it should be done now, instead of later